system76/coreboot
# mb/system76: Configure I2C HID IRQs as level triggered

Upstream: [CB:61343](https://review.coreboot.org/c/coreboot/+/61343)

Per Microsoft's spec for [HID over I2C](http://download.microsoft.com/download/7/d/d/7dd44bb7-2a7a-4505-ac1c-7227d3d96d5b/hid-over-i2c-protocol-spec-v1-0.docx), interrupts must be level triggered. Switch GPIOs and the devicetree config to conform to this.

This affects

- [ ] addw1
- [ ] addw2
- [ ] bonw14
- [ ] darp5
- [ ] darp6
- [ ] gaze14
- [x] lemp9 - Tested by @jackpot51 
- [x] oryp6 - Tested by @crawfxrd 
- [x] oryp7 - Tested by @jacobgkau 

## How to test

- Blacklist PS/2 driver (boot with `modprobe.blacklist=psmouse`)
- Check dmesg for logs about touchpad device
- Check `/proc/bus/input/devices` for detected touchpad device
- Use `sudo libinput debug-gui` to check multitouch
